flake8diff/main.py

```diff
@@ -114,10 +113,33 @@
          'Can be any of "{0}"'
          ''.format(', '.join(STRICT_MODES.keys())),
 )
+parser.add_argument(
+    '--version',
+    action='store_true',
+    default=False,
+    help='Show the version number of flake8-diff'
+)
+
+
+def version():
+    msg = (
+        'flake8-diff\n'
+        '===========\n'
+        '{}\n\n'
```

Comment:
I'm not aware of any standards, but I wouldn't put the name and description in `--version`. Chances are high that people who use `--version` probably know what flake8-diff is already, and they're specifically looking for just the version.

Having the python version is a nice touch though.